website/src/pages/extensions/index.tsx

```
@@ -1,24 +1,682 @@
+                        setTimeout(() => {
+                          setReadmeState(prev => ({ ...prev, copyFeedback: false }));
+                        }, 2000);
+                      } catch (err) {
+                        console.error('Failed to copy permalink:', err);
+                      }
+                    };
+                    copyToClipboard().catch((err: unknown) => {
+                      console.error('Failed to copy permalink:', err);
+                    });
+                  }}
+                  className="w-full bg-gray-100 dark:bg-charcoal-600 text-gray-700 dark:text-gray-300 py-2 px-4 rounded-lg hover:bg-gray-200 dark:hover:bg-charcoal-500 transition-colors flex items-center justify-center cursor-pointer">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)